app/views/developers/show.html.erb

```diff
@@ -88,13 +88,42 @@
             </div>
           </div>
     </div>
-  </div>
+      <div class="row">
+        <div class="bio col-8">
+          <h5><strong>Bio:</strong></h5> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Cum maxime aperiam, laudantium quibusdam laboriosam iste reprehenderit. Numquam, nobis quaerat adipisci voluptatum omnis. Saepe illo quasi sint iusto consequuntur vero labore. <%= @developer.bio %></p>
+        </div>
+      </div>
+      <% if @hired_jobs.present? %>
 
-  <div class="row">
-    <div class="bio col-8">
-      <p><strong>Bio:</strong> <%= @developer.bio %></p>
-    </div>
+       <% if current_user && @hired_jobs.find { |job| job.company == current_user.company } %>
+        <%= link_to 'Add review', new_application_review_path(job.applications.where(developer: @developer).first) %>
```

Comment:
I get this error on this line of code when I try to view a developers show page:

![Screen Shot 2020-03-16 at 3 23 41 PM](https://user-images.githubusercontent.com/57788075/76732676-3d57a500-679a-11ea-93f0-86acdc92eb38.png)